### ISSUE TITLE
Button field

### ISSUE THREAD

**Issue by paulyili** [twentyhq/twenty#4335]
### Feature:
Button field for tables

### Use Case:
If you're using the CRM to track customers from an ecommerce platform, you can set a URL on the button to link it to the customers profile on the ecommerce site. or show their order list. etc. 

### Functionality

- Set button label. E.g "Customer profile" or "current orders" etc
- Set button colour, blue, green, etc. 
- right click to edit button url (for manual set)
- set URL via API.

**Comment by FelixMalfait** [comment]
Yes great idea and this is planned! We did a first hardcoded proof of concept called "quick actions" but we will remove everything and redo it from scratch. It will not be a field types but a new concept called "commands" (they would also be displayed in the command bar).

I'd say this will ship around end of Q2. It's not part of our core roadmap but something we could likely tackle on the side with community help!

**Comment by Bonapara** [comment]
Would the ability to click links in a `text field` work for you too? (We also plan to implement this feature)

**Comment by paulyili** [comment]
> Would the ability to click links in a `text field` work for you too? (We also plan to implement this feature)

Of course! that would be useful for the domain names etc. 
the "button" was more of a clean approach.

**Comment by giovannicocco** [comment]
@FelixMalfait If I wanted to have an action button to send a message via WhatsApp using WhatsApp API Business, would it be through this new command concept or do you have another suggestion?

**Comment by FelixMalfait** [comment]
@giovannicocco sorry for the late reply, I missed that message. Yes exactly we would be able to support input to.
